helm/libero-reviewer/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "libero-reviewer.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+    helm.sh/chart: {{ include "libero-reviewer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  type: {{ .Values.service.type }}
```

Comment:
Maybe we should hardcore this because we only support LoadBalancer as a value?

Reply:
/cc @erezmus opinion?

Reply:
probably